src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -205,14 +205,19 @@ void FrontEnd::translate_graph(const ov::frontend::InputModel::Ptr& model,
 
         if (port_type == "none") {
             for (const auto& node_output : ng_op_map[operation_name]) {
-                results.push_back(std::make_shared<ov::opset8::Result>(node_output));
+                auto result_node = std::make_shared<ov::opset8::Result>(node_output);
+                // for alignment with legacy TensorFlow frontend, we must use OpName:0 for Result node
+                set_node_name(operation_name + ":" + std::to_string(port_index), result_node);
```

Comment:
I believe we need both names with ":0" and without.
How is it actually implemented in tf framework? do they have this ":0" for tensors?

Reply:
Generally, now it does not affect API 2.0 inference because it does not rely on Result node name at all. I just set more meaningful name for it